### ISSUE TITLE
Restart work on rst2pdf

### ISSUE THREAD

**Issue by pmaupin** [rst2pdf/rst2pdf#538]
So, here's where we figure out some ground rules for getting this thing going again.

Currently, there are 3 members of the rst2pdf team -- Roberto (who probably wants to shut off notifications if he hasn't already done so :), me, and aquavitae if he wants the job.  aquavitae has already put a lot of work into a 2-to-3 conversion on his own --unfortunately, his starting point was the google repo, not Roberto's, so I have rebased his code and added it to the repo on a separate branch.

Anyway, if anybody else wants write access to the repo to make keeping in sync easier, just drop me a note.

I think the right answer for moving the master branch forward is to:

a) do the usual github thing of always having one reviewer vet the changes;
b) first get all the existing tests working on it with Python 2.7, travisci and some fairly recent libraries, such as reportlab 3.2, etc., before starting to add functional changes;
c) require test cases for functional changes

Questions and comments are more than welcome.

Thanks,
Pat

**Comment by lornajane** [comment]
Gonna close this because the project is currently _more_ active (more hands always welcome!) and the tests are in great shape. Even the github reviewing thing is in place too - this was a great list to get a project restarted :)
